src/templates/fragments/desc.html

```diff
@@ -1,10 +1,3 @@
-  <input
-    class="text-lg col-span-3 px-1.5 py-3.5 border border-gray-800 bg-gray-400 bg-opacity-75 placeholder-blue-800 placeholder-opacity-75 rounded-md"
-    name="name"
-    type="text"
-    value="{{item.name}}"
-    placeholder="{{localize 'swnr.sheet.name'}}"
-  />
```

Comment:
Refactoring this is out of scope for this PR.

Reply:
Open @ #38.